app/services/katello/pulp3/content_view_version.rb

```diff
@@ -0,0 +1,50 @@
+module Katello
+  module Pulp3
+    class ContentViewVersion
```

Comment:
Having all this code in one place makes a lot of sense to me. Thanks for the refactor!

Reply:
Great suggestion!